source/vibe/core/args.d

```diff
@@ -304,7 +304,7 @@ private T fromValue(T)(in JSONValue val)
 	static if (is(T == bool)) return val.type == JSON_TYPE.TRUE;
 	else static if (is(T : long)) return val.integer.to!T;
 	else static if (is(T : double)) return val.floating.to!T;
-	else static if (is(T == string)) return val.str;
+	else static if (is(T == string)) return (val.type == JSONType.string) ? val.str : val.to!T;
```

Comment:
Isn't it JSON_TYPE?

Reply:
Have a look at https://travis-ci.org/vibe-d/vibe-core/jobs/520577020 (this is with the very old 2.078.3)

```
source/vibe/core/args.d(307,55): Error: undefined identifier JSONType
```

Reply:
As mentioned before, this is due to this change: https://github.com/dlang/phobos/pull/6649

Reply:
Thanks.